llvm/include/llvm/DWARFLinker/Classic/DWARFLinkerDeclContext.h

```diff
@@ -115,6 +118,7 @@ class DeclContext {
   uint16_t Tag = dwarf::DW_TAG_compile_unit;
   unsigned DefinedInClangModule : 1;
   StringRef Name;
+  StringRef NameForUniquing;
```

Comment:
Nit: can you add a comment here to explain the difference? `NameForUniquing` is self-explanatory, but may leave you wondering what `Name` is for.